express.js

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable */
+
 const express = require('express');
 
 const app = express();
 const PORT = 3000;
```

Comment:
```suggestion
const PORT = 3000;
const port = process.env.PORT || PORT;
```
Полезно иметь возможность прокинуть порт снаружи